CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ clcache changelog
 ## Upcoming release
 
  * Bugfix: Supportuse of py2exe with Python 3.4
+ * Bugfix: Support includes parsing when cl language is not Englich (GH #65)

```

Comment:
Right, thanks. Could you merge #66 first? Then we don't get merge conflicts in this part.